modules/RTC/RTC.js

```diff
@@ -542,16 +541,33 @@ export default class RTC extends Listenable {
 
     /**
     * Set mute for all local video streams attached to the conference.
-    * @param value The mute value.
     * @returns {Promise}
     */
-    setVideoMute(value) {
+    setVideoMute() {
         const mutePromises = [];
 
         this.getLocalTracks(MediaType.VIDEO)
             .forEach(videoTrack => {
                 // this is a Promise
-                mutePromises.push(value ? videoTrack.mute() : videoTrack.unmute());
+                mutePromises.push(videoTrack.mute());
+            });
+
+        // We return a Promise from all Promises so we can wait for their
+        // execution.
+        return Promise.all(mutePromises);
+    }
+
+    /**
+    * Set mute for all local desktop video streams attached to the conference.
+    * @returns {Promise}
+    */
+    setDesktopMute() {
+        const mutePromises = [];
+
+        this.getLocalTracks(MediaType.DESKTOP)
```

Comment:
I just remembered that the screen share tracks have their type set to `video`, `videoType` is the one that  distinguishes them from camera tracks. So this would always return an empty array.

Also, do we want `setVideoMute` to mute only camera tracks and not the desktop tracks? If so, `setVideoMute` also needs to be modified.

Reply:
Updated, PTAL!